### PR TITLE
Added band annotation documentation

### PR DESCRIPTION
**Pull request by ColinEberhardt**

### CODE REVIEW

File: site/src/components/annotation/band.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      .call(verticalBand);
+
+  var horizontalBand = fc.annotation.band()
+      .xScale(xScale)
+      .yScale(yScale)
+      .y0(function(d) { return d[0]; })
+      .y1(function(d) { return d[1]; });
+
+  var ticks = yScale.ticks();
+  var yBands = d3.zip(ticks, ticks.slice(1))
```
Comment (chrisprice) [inline site/src/components/annotation/band.md]: `d3.pair`?

File: site/src/components/annotation/line.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: component
-title: Line Annotation
+title: Line
 component: tool/line.js
```
Comment (chrisprice) [inline site/src/components/annotation/line.md]: `annotation/line.js`

**Comment by chrisprice** [comment]
Just a couple of comments.

**Comment by ColinEberhardt** [comment]
Thanks @chrisprice updated and merged

**Pull request merged**